packages/sdks/snippets/qwik-city/src/routes/live-preview/[...index]/index.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  component$,
+  useSignal,
+  useTask$,
+  useVisibleTask$,
+} from '@builder.io/qwik';
+import { routeLoader$ } from '@builder.io/qwik-city';
+import type { BuilderContent } from '@builder.io/sdk-qwik';
+import {
+  fetchOneEntry,
+  getBuilderSearchParams,
+  subscribeToEditor,
+} from '@builder.io/sdk-qwik';
+
+export const useBuilderContent = routeLoader$(async ({ url }) => {
+  const data = await fetchOneEntry({
+    model: 'blog-data',
+    apiKey: 'ee9f13b4981e489a9a1209887695ef2b',
+    options: getBuilderSearchParams(url.searchParams),
+    userAttributes: {
+      urlPath: url.pathname,
+    },
+  });
+  return data ?? null;
+});
+
+export default component$(() => {
+  const { value } = useBuilderContent();
```

Comment:
destructuring the signal like that can lead to issues and is not typical syntax.

You usually want to do
```suggestion
  const builderContent = useBuilderContent();
  
  // somewhere later, when you need to access it
  builderContent.value
```

Reply:
@samijaber This was the original approach, but I thought I could refactor this a bit. Thanks for the suggestion, I'm working on Qwik for the first time. I have reverted.